pom.xml

```diff
@@ -162,4 +164,59 @@
       </pluginRepository>
     </pluginRepositories>
 
+    <profiles>
+        <profile>
+            <id>license</id>
+            <build>
+                <defaultGoal>license:format</defaultGoal>
+                <plugins>
+                    <plugin>
+                        <groupId>com.mycila</groupId>
+                        <artifactId>license-maven-plugin</artifactId>
+                        <version>${mycila-license.version}</version>
+                        <configuration>
+                            <header>header.txt</header>
```

Comment:
This setup is going to work only when Maven is called from a directory where license.txt is located. I guess you are aware of that and that you decided to ignore the drawbacks, right?

Reply:
is part of the dependency classpath

Reply:
True, thanks for the explanation.